### PR TITLE
adds pea wine

### PR DESCRIPTION
**Pull request by boodaliboo**
you can now distill peas into pea wine
previously it didnt create anything
#### Changelog

:cl:  
rscadd: Added pea wine and a way to distill it
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2304,3 +2304,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
 	glass_name = "Godmother"
 	glass_desc = "A lovely fresh smelling cocktail, a true Sicilian delight."
+
+/datum/reagent/consumable/ethanol/peewine
+	name = "Pea Wine"
+	description = "An alcoholic beverage That is created through distilling pees."
+	color = "#008000" // rgb: 0, 128, 0
+	nutriment_factor = 1 * REAGENTS_METABOLISM
+	boozepwr = 25
+	taste_description = "rotting vegatables"
```
Comment (patpol4) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: vegetables

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -2304,3 +2304,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	glass_name = "glass of pea wine"
+	glass_desc = "A freezing glass of pea wine."
```
Comment (patpol4) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2316]: Why is it cold

Comment (boodaliboo) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2316]: i dunno

Comment (patpol4) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2316]: maybe its the peas

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -2304,3 +2304,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "godmother"
 	glass_name = "Godmother"
 	glass_desc = "A lovely fresh smelling cocktail, a true Sicilian delight."
+
+/datum/reagent/consumable/ethanol/peewine
+	name = "Pea Wine"
+	description = "An alcoholic beverage That is created through distilling pees."
```
Comment (jcatjmeow) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	description = "An alcoholic beverage That is created through distilling peas."
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

**Pull request merged**